### PR TITLE
feat(tensorflow frontend): Added a new inverse window function

### PR DESCRIPTION
**Pull request by teckno**
https://github.com/unifyai/ivy/issues/21889

# PR Description

## Related Issue

Close #

## Checklist

- [x] Did you add a function?
- [x] Did you add the tests?
- [x] Did you run your tests and are your tests passing?
- [x] Did pre-commit not fail on any check?
- [x] Did you follow the steps we provided?

### CODE REVIEW

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_signal.py [hunk 1]
```diff
@@ -53,6 +53,39 @@ def _valid_stft(draw):
 # ------------ #
 
 
+# inverse_stft_window_fn
+@handle_frontend_test(
+    fn_tree="ivy.signal.inverse_stft_window_fn",
```
Comment (hirwa-nshuti) [inline ivy_tests/test_ivy/test_frontends/test_tensorflow/test_signal.py:58]: This should be the frontend function not `Ivy` one 😅

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_signal.py [hunk 2]
```diff
@@ -53,6 +53,39 @@ def _valid_stft(draw):
...
+    ),
+    test_with_out=st.just(False),
+)
+def test_inverse_stft_window_fn(
+    *, dtype_and_x, test_flags, backend_fw, fn_tree, on_device, frontend
+):
+    input_dtype, x = dtype_and_x
+
+    # Create a function to test
+    def inverse_stft_window_fn(frame_step):
```
Comment (hirwa-nshuti) [inline ivy_tests/test_ivy/test_frontends/test_tensorflow/test_signal.py:73]: Would you clearly explain the use of this logic 🙄

File: ivy/functional/frontends/tensorflow/signal.py [hunk 3]
```diff
@@ -19,6 +19,22 @@ def idct(input, type=2, n=None, axis=-1, norm=None, name=None):
     return ivy.dct(input, type=inverse_type, n=n, axis=axis, norm=norm)
 
 
+@to_ivy_arrays_and_back
+def inverse_stft_window_fn(frame_step, forward_window_fn=ivy.hann_window, name=None):
```
Comment (hirwa-nshuti) [inline ivy/functional/frontends/tensorflow/signal.py:23]: `forward_window_fn` should be a frontend version of the mentioned function

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hey @teckno 

Check the comments below

**Comment by Sarvesh-Kesharwani** [comment]
Closing the PR for the time being, since there has been no activity for more than 2 weeks.

**Pull request closed without merging**